src/components/vr-accessibility/vr-accessibility.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, ElementRef, Input, OnChanges, OnDestroy, OnInit, SimpleChanges, ViewChild } from '@angular/core';
+import { Hapticplot } from '../../d3/hapticplot.d3';
+import 'aframe';
+import 'aframe-extras';
+import 'super-hands';
+import 'aframe-haptics-component';
+
+
+
+@Component({
+  selector: 'app-vr-accessibility',
+  templateUrl: './vr-accessibility.component.html'
+})
+export class VRAccessibilityComponent implements OnInit, OnChanges, OnDestroy{
+  private vrHapticPlot: Hapticplot;
+  private shape: string;
+  private color: string;
+
+  constructor(
+  ) {
+    this.vrHapticPlot = new Hapticplot('a-sphere');
```

Comment:
You can put this on line 15 where the property is declared. Empty constructors are best constructors.